packages/contracts/src/systems/FacilitySystem.sol

```diff
@@ -79,7 +110,7 @@ contract FacilitySystem is System {
    */
   function destoryFacility(bytes32 entityKey) public {
```

Comment:
English spelling nit: 
`destory` should be `destroy`

Reply:
Thanks. Good catch.
Fixed.